resources/i18n/en.json

```diff
@@ -97,7 +97,9 @@
     "open-in-browser":"Open in browser",
     "start-kiwix-server":"Start Kiwix Server",
     "stop-kiwix-server":"Stop Kiwix Server",
-    "all":"All",
+    "all":"All - Dual Stack Mode",
```

Comment:
Such text contradicts the corresponding entry in `qqq.json` (which reads `{{Identical|All}}`). There will be a problem for `gt("all")` used in other context (for example, for filtering the library by category). If we want a more elaborate text for IPs we should introduce a new translation entry (e.g. `all_ips`).